### ISSUE TITLE
vi autosuggestion added to end of vi command

### ISSUE THREAD

**Issue by kkanungo17** [zsh-users/zsh-autosuggestions#191]
Whenever I type a new command starting with vi, the file path has a 'vi' suggestion appended to the end.

![screen shot 2016-08-27 at 22 08 40](https://cloud.githubusercontent.com/assets/11506039/18028681/da73051a-6ca2-11e6-8180-cbfb3218b0c5.png)

**Comment by jmduke** [comment]
I run into the same thing with `cd`.

**Comment by ericfreese** [comment]
That definitely shouldn't be happening. What version of this plugin are you using? What version of zsh? Do you get the same behavior when running *only* the z-asug plugin (no additional zshrc configuration)?

```shell
% zsh -f
%% source zsh-autosuggestions.zsh
```

**Comment by jmduke** [comment]
```
[jmduke:~]$ zsh --version
zsh 5.2 (x86_64-apple-darwin16.0)
[jmduke:~]$ echo ~/.zsh/zsh-autosuggestions/VERSION
/Users/jmduke/.zsh/zsh-autosuggestions/VERSION
```

I am unable to repro just using the plugin -- but the only other two plugins I use are `git` and `python`.

![screenshot 2016-10-01 18 21 43](https://cloud.githubusercontent.com/assets/461046/19017919/047b4dc8-8804-11e6-99e3-3b4da5ad9d5d.png)

Happy to provide any more details that may be of use!

**Comment by kkanungo17** [comment]
Yup, works perfectly if only the plugin is used. Maybe the order of my plugins screwed it up?
![screen shot 2016-10-02 at 18 10 52](https://cloud.githubusercontent.com/assets/11506039/19020614/9fb88950-88cb-11e6-81c4-42b31b1f27dd.png)

**Comment by subho007** [comment]
https://github.com/zsh-users/zsh-autosuggestions/pull/198 fixes this problem

**Comment by ericfreese** [comment]
Apologies, I should have noticed this earlier, but this is a duplicate of #168, which will be fixed with v0.3.3 (#169).

In the mean time, before v0.3.3 is released, you can use the `develop` branch instead of `master`.

**Comment by jmduke** [comment]
Confirming that switching to develop solved my issue :) Thanks so much for the help!
